Terminal.Gui/Drawing/Aligner.cs

```diff
@@ -1,40 +1,75 @@
+using System.ComponentModel;
+using Microsoft.CodeAnalysis;
+using static Terminal.Gui.Pos;
+
 namespace Terminal.Gui;
 
 /// <summary>
-///     Controls how the <see cref="Justifier"/> justifies items within a container. 
+///     Controls how the <see cref="Aligner"/> aligns items within a container.
 /// </summary>
-public enum Justification
+public enum Alignment
 {
     /// <summary>
     ///     The items will be aligned to the left.
-    ///     Set <see cref="Justifier.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one space between
+    ///     Set <see cref="Aligner.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one space between
     ///     each item.
     /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///         If the container is smaller than the total size of the items, the right items will be clipped (their locations
+    ///         will be greater than the container size).
+    ///     </para>
+    /// </remarks>
     /// <example>
     ///     <c>
     ///         111 2222 33333
     ///     </c>
     /// </example>
     Left,
 
+    /// <summary>
+    ///     The items will be aligned to the top.
+    ///     Set <see cref="Aligner.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one line between
+    ///     each item.
+    /// </summary>
+    Top = Left,
+
     /// <summary>
     ///     The items will be aligned to the right.
-    ///     Set <see cref="Justifier.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one space between
+    ///     Set <see cref="Aligner.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one space between
     ///     each item.
     /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///         If the container is smaller than the total size of the items, the left items will be clipped (their locations
+    ///         will be negative).
+    ///     </para>
+    /// </remarks>
     /// <example>
     ///     <c>
     ///         111 2222 33333
     ///     </c>
     /// </example>
     Right,
 
+    /// <summary>
+    ///     The items will be aligned to the bottom.
+    ///     Set <see cref="Aligner.PutSpaceBetweenItems"/> to <see langword="true"/> to ensure at least one line between
+    ///     each item.
+    /// </summary>
+    Bottom = Right,
```

Comment:
Why are dissimilar labels sharing values?

There are several reasons every other existing framework uses separate enumerations for vertical and horizontal alignment.